### PR TITLE
Spelling error fixes

### PR DESCRIPTION
**Pull request by karinbredenberg**
Spelling error fixes and rewording during the last month.

Various issues handled. But no issues will be closed for now, more changes might be needed.

### CODE REVIEW

File: profile/CSIP.xml [hunk 1]
```diff
@@ -154,7 +154,7 @@
             <requirement ID="CSIP1" REQLEVEL="MUST" EXAMPLES="metsRootElementExample1 metsRootElementExample2 metsRootElementExample3">
                 <description>
                     <head>Content Identification</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">It is mandatory to use a content ID which is expressed with @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">It is mandatory to use a content identification in the form of a string which is expressed within the attribute @OBJID. The value should be the same as the name or ID of the package (the name of the root folder) for the root METS document or the name and folder name for the representation. The OBJID must meet the principle of being unique at least across the repository.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: It is mandatory to use the @OBJID attribute as a string identifier for the package. The value should be identical to the name/ID of the package, i.e. the name of the root folder, for the root METS document or the name and folder name for a representation.

File: profile/CSIP.xml [hunk 2]
```diff
@@ -350,7 +350,7 @@
             <requirement ID="CSIP20" REQLEVEL="SHOULD" RELATEDMAT="VocabularyStatus" EXAMPLES="dmdSecExample1">
                 <description>
                     <head>Status of the descriptive metadata</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">Status of the metadata. Used to indicate the currency of the package. If used the two values “SUPERSEDED” or “CURRENT” from the vocabulary is used.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">Status of the metadata. Used to indicate the currency of the package using the terms “SUPERSEDED” or “CURRENT”.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: Indicates the status of the package using a fixed vocabulary.

File: profile/CSIP.xml [hunk 3]
```diff
@@ -785,7 +785,7 @@
             <requirement ID="CSIP62" REQLEVEL="SHOULD" RELATEDMAT="VocabularyContentInformationTypeSpecification" EXAMPLES="fileSecExample1 fileSecExample2">
                 <description>
                     <head>Specific content type</head>
-                    <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory to use when the file group catagorization is Representations. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
+                    <p xmlns="http://www.w3.org/1999/xhtml">An added attribute which describes the specific content information type specification used for the transferred content. The attribute is mandatory to use when the file group catagorisation is Representations. The vocabulary is going to evolve under the care of the DILCIS Board as additional content information type specifications are developed.</p>
```
Comment (carlwilson) [inline profile/CSIP.xml]: The attribute is mandatory when the file group catagorisation value is "Representations". The vocabulary will evolve under the curation of the DILCIS Board as additional content information type specifications are developed.

File: profile/CSIP.xml (hunk 1 above)
Comment (karinbredenberg) [reply profile/CSIP.xml]: Fixed.

File: profile/CSIP.xml (hunk 2 above)
Comment (karinbredenberg) [reply profile/CSIP.xml]: Fixed

File: profile/CSIP.xml (hunk 3 above)
Comment (karinbredenberg) [reply profile/CSIP.xml]: Fixed.

**Pull request merged**